### PR TITLE
DRY-ing out.

### PR DESCRIPTION
**Pull request by akash1810**
Looks scary as its kind of self referential... it compiles though :dancers:

### CODE REVIEW

**Comment by akash1810** [comment]
Future TODO:
- [ ] do the same for [all strings](https://github.com/guardian/grid/blob/master/common-lib/src/main/scala/com/gu/mediaservice/model/UsageRights.scala#L33-L50).

File: common-lib/src/main/scala/com/gu/mediaservice/model/UsageRights.scala [hunk 1]
```diff
@@ -60,24 +60,24 @@ object UsageRights {
...
-        case _                           => None
+        case "chargeable"                       => json.asOpt[Chargeable]
+        case "agency"                           => json.asOpt[Agency]
+        case "commissioned-agency"              => json.asOpt[CommissionedAgency]
+        case "PR Image"                         => json.asOpt[PrImage]
+        case "handout"                          => json.asOpt[Handout]
+        case "screengrab"                       => json.asOpt[Screengrab]
+        case "guardian-witness"                 => json.asOpt[GuardianWitness]
+        case "social-media"                     => json.asOpt[SocialMedia]
+        case "obituary"                         => json.asOpt[Obituary]
```
Comment (jamesgorrie) [inline common-lib/src/main/scala/com/gu/mediaservice/model/UsageRights.scala]: Could we not do all of them while we're here?

Comment (akash1810) [inline common-lib/src/main/scala/com/gu/mediaservice/model/UsageRights.scala]: Yep...though now theres loads of whitespace changes in the diff :crying_cat_face:

**Comment by jamesgorrie** [comment]
Fabulous :+1:

**Comment by theefer** [comment]
Nice, makes uses a lot more discoverable at compilation-time/type-level.

**Pull request merged**